app/js/reducers/orderSelectionReducer.js

```diff
@@ -0,0 +1,18 @@
+import { SET_SELECTED_ORDER } from '../actions/actionTypes';
+import initialState from './initialState';
+
+const orderSelectionReducer = (state = initialState.orderSelection, action) => {
+  switch (action.type) {
+    case SET_SELECTED_ORDER:
+      return {
+        ...state,
+        currentOrderType: action.currentOrderType,
+        selectedOrder: action.order,
+        activity: action.activity,
+      };
```

Comment:
why don't you just `destructure` instead of repeating `action` in these three instances..

Reply:
done!